src/unused_finder/unused_finder.rs

```diff
@@ -133,54 +122,55 @@ impl UnusedFinder {
         Ok(Self {
             config,
             entry_packages,
-            tsconfig,
             skipped_dirs,
             skipped_items,
-            flattened_walk_file_data,
+            resolver: Some(resolver),
             graph,
             file_path_exported_items_map,
             ..Default::default()
         })
     }
 
+    // Read and parse all files from disk have a fresh in-memory representation of self.entry_files and self.graph information
     #[napi]
     pub fn refresh_file_list(&mut self) {
-        let resolver = create_caching_resolver(&self.tsconfig);
-        self.flattened_walk_file_data = create_flattened_walked_files(
+        // Get a vector with all WalkFileMetaData
+        let mut flattened_walk_file_data = create_flattened_walked_files(
             &self.config.paths_to_read,
             &self.skipped_dirs,
             &self.skipped_items,
         );
-        self.file_path_exported_items_map = self
-            .flattened_walk_file_data
-            .clone()
-            .par_drain(0..)
-            .map(|file| {
-                let ids = file
-                    .import_export_info
-                    .exported_ids
-                    .iter()
-                    .map(|exported_item| ExportedItemReport {
-                        id: exported_item.metadata.export_kind.to_string(),
-                        start: exported_item.metadata.span.lo.to_usize() as i32,
-                        end: exported_item.metadata.span.hi.to_usize() as i32,
-                    })
-                    .collect();
-                (file.source_file_path.clone(), ids)
+        // Proccess all information with swc resolver to resolve symbols within tsconfig.paths.json
```

Comment:
"resolve symbols" typically refers to names, e.g. symbols within an object file. I _dont'_ think that is what's happening here, since the swc resolver is for resolving import paths. What exactly is going on here?